Vladyslav_Gurevych/HW7/Task4.py

```diff
@@ -0,0 +1,19 @@
+import re
+
+
+def remove_numbers_from_file(text_with_digits: list) -> list:
```

Comment:
it is better to have file_path as a parameter...and open it inside function then return string without digits

Reply:
+